ajax/libs/lamejs/package.json

```diff
@@ -0,0 +1,27 @@
+{
+  "name": "lamejs",
+  "filename": "lame.min.js",
+  "description": "Pure JavaScript MP3 Encoder",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/zhuker/lamejs.git"
```

Comment:
I suggest to use  "url": "git://github.com/zhuker/lamejs.git"

Reply:
@sashberd this is fine.
We can let this as the same with upstream https://github.com/zhuker/lamejs/blob/master/package.json#L15
Also there are some lib use this pattern, such as https://github.com/cdnjs/cdnjs/blob/master/ajax/libs/tslib/package.json#L20

Thanks.